openlibrary/templates/check_ins/reading_goal_progress.html

```diff
@@ -18,14 +19,29 @@
           <span class="reading-goal-progress__completed" style="width: $(completed)%;"></span>
         </div>
         <div class="reading-goal-progress__details">
-          $:_('<span class="reading-goal-progress__books-read">%(books_read)d</span>/<span class="reading-goal-progress__goal">%(goal)d</span> books', books_read=goal.books_read, goal=goal.goal) <span class="reading-goal-progress__percentage">($goal.progress%)</span>
+          $:_('<span class="reading-goal-progress__books-read">%(books_read)d</span>/<span class="reading-goal-progress__goal">%(goal)d</span> books', books_read=count, goal=goal.goal) <span class="reading-goal-progress__percentage">($goal.progress%)</span>
           <a class="edit-reading-goal-link" href="javascript:;">$_('Edit')</a> or
 	  <a href="https://blog.openlibrary.org/2022/12/31/reach-your-2023-reading-goals-with-open-library">$_('Learn more')</a>
         </div>
       </div>
+      <div>
+        $(goal.goal)
+        <br>
+        $(goal.year)
+        <br>
+        $(goal.books_read)
+      </div>
       $ reading_goal_form = render_template('check_ins/reading_goal_form', goal=goal.goal, year=goal.year, update=True)
       $ id = 'yearly-goal-modal-%d' % loop.index
       $ title = _('Edit %(year)d Reading Goal', year=goal.year)
       $:render_template('native_dialog', id, reading_goal_form, title=title)
     </div>
 </div>
+
+
+<script>
+
+  function check() {
+      alert(goal);
+  }
```

Comment:
The ⛔ below is because the GitHub Editor expects each text file to end in one and only one carriage return (`\n`).  I think `pre-commit` should autofix it but it is a good habit to get into.

Reply:
Where should I find ⛔️ ? sorry but I didn't quite understand what you meant...

Reply:
Click https://github.com/internetarchive/openlibrary/pull/7626/files#r1132689117 and then
below ⬇️ ⬇️ ⬇️   Maybe the emoji only happens in firefox.

Reply:
haha okay I see. Is using Mozilla Firefox motivated by the reason it is open source ? and the Richard Stallman ideology ?

Reply:
Nope.  Just because it works consistently.

Reply:
@scottbarnes @cclauss I need help about why javascript_tests failing.

Reply:
@gobbledy-gook, that failure wasn't your fault. I've submitted a PR to add some JavaScript tests to bring the percentage of JavaScript code back over the 16% threshold.